Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -0,0 +1,278 @@
+                guard let featureTable = feature.table as? ServiceFeatureTable,
+                      let symbol = featureTable.layerInfo?.drawingInfo?.renderer?.symbol(for: feature) else {
+                    return nil
+                }
+                return Graphic(geometry: feature.geometry, symbol: symbol)
+            }
+            return graphics
+        }
+        
+        /// Creates the associations graphics for elements in the container.
+        /// - Parameter boundingExtent: The boundary for the container.
+        /// - Returns: An array of association relationship graphics.
+        private func makeGraphicsForAssociations(within boundingExtent: Envelope) async throws -> [Graphic] {
+            let containmentAssociations = try await network.associations(forExtent: boundingExtent)
+            let graphics: [Graphic] = containmentAssociations.map { association in
+                let symbol = association.kind == .attachment ? attachmentSymbol : connectivitySymbol
+                return Graphic(geometry: association.geometry, symbol: symbol)
+            }
+            return graphics
+        }
+        
+        // MARK: Helpers
+        
+        /// Changes the visibility of the operational layers.
+        /// - Parameter isVisible: A Boolean to make the map visible or not.
+        func setOperationalLayersVisibility(isVisible: Bool) {
+            map.operationalLayers.forEach { $0.isVisible = isVisible }
+        }
+        
+        /// Sets status message.
+        /// - Parameter message: A message of the current status.
+        @MainActor
+        private func setStatusMessage(_ message: String) {
+            statusMessage = message
+        }
+    }
+}
+
+private extension Item {
+    /// A web map portal item for the Naperville Electric Containers.
+    static func napervilleElectricalNetwork() -> PortalItem {
+        PortalItem(
+            // Sample server 7 authentication required.
+            portal: Portal(url: .samplePortal, connection: .authenticated),
+            id: .init("813eda749a9444e4a9d833a4db19e1c8")!
+        )
+    }
+}
+
+private extension URL {
+    /// The server containing the data for this sample.
+    private static var sampleServer7: URL {
+        URL(string: "https://sampleserver7.arcgisonline.com")!
+    }
+    
+    /// The feature service containing the data for this sample.
+    static var featureService: URL {
+        sampleServer7.appendingPathComponent("server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")
+    }
+    
+    /// The portal containing the data for this sample.
+    static var samplePortal: URL {
+        sampleServer7.appendingPathComponent("portal")
+    }
+}
+
+/// A struct for displaying legend info in a list row.
+struct LegendItem {
+    /// The description label of the legend item.
+    let name: String
+    
+    /// The image swatch of the legend item.
+    let image: UIImage
```

Comment:
We usually don't put blank lines between fields of a struct.

```suggestion
    /// The description label of the legend item.
    let name: String
    /// The image swatch of the legend item.
    let image: UIImage
```